ISSUE_TEMPLATE

```diff
@@ -0,0 +1,8 @@
+### What went wrong?
+
+
+### How do we reproduce?
+
+
+### Component versions (python-gssapi, Kerberos, OS, etc.)
```

Comment:
`Kerberos distribution (MIT, Heimdal, etc) and version`, perhaps (so that we don't have to guess Kerberos distro by version number formatting)?